### PR TITLE
Fix #1115 make possible to crouch with 1hp

### LINKED ISSUE #1115: Players can't crouch when they have 1 HP
**Describe the bug**
You can't crouch when you set your HP to 1.

**To reproduce**
1. run toggleControl(getPlayerFromName("MrDadosz"), "crouch", true)
2. run setElementHealth(getPlayerFromName("MrDadosz"), 100)
now you can crouch
3. run setElementHealth(getPlayerFromName("MrDadosz"), 1)
now you can't crouch
4. run toggleControl(getPlayerFromName("MrDadosz"), "crouch", true)
you can't crouch
5. run setElementHealth(getPlayerFromName("MrDadosz"), 2)
now you can crouch

**Expected behaviour**
Player should can crouch everytime, even when he has 1 HP.

**Screenshots**
https://youtu.be/u4dZs_IERDs

**Version**
Multi Theft Auto v1.5.7-release-20288
MTA:SA Server v1.5.6-release-16609 and MTA:SA Server v1.5.7-release-18957 - same results

### PR DESCRIPTION
**Pull request by CrosRoad95**
It just giving ability to crouching with 1hp, fix for #1115 
image

### CODE REVIEW

File: Client/multiplayer_sa/CMultiplayerSA.cpp [hunk 1]
```diff
@@ -1486,6 +1486,12 @@ void CMultiplayerSA::InitHooks()
     MemSet((void*)0x72925D, 0x1, 1);            // objects
     MemSet((void*)0x729263, 0x1, 1);            // players
 
+    
+    // Allow crouching with 1HP
+    float fDuckingHealthThreshold;
+    MemPut((void*)0x6943AD, &fDuckingHealthThreshold);
```
Comment (ccw808) [inline Client/multiplayer_sa/CMultiplayerSA.cpp:1492]: `&fDuckingHealthThreshold` will be a memory location on the stack. Is that intended?

Comment (CrosRoad95) [reply Client/multiplayer_sa/CMultiplayerSA.cpp:1492]: i had to do it in that way due it refer to global float number
0x6943AB: `D8 1D 24 86 85 00                       fcomp   ds:fOne         ; 1.0f`
and that variable is used in 2855 other locations what cause i just can't do simple `memset(0x6943AD, 0)`

Comment (ccw808) [reply Client/multiplayer_sa/CMultiplayerSA.cpp:1492]: `float fDuckingHealthThreshold` should not be a local variable

Comment (CrosRoad95) [reply Client/multiplayer_sa/CMultiplayerSA.cpp:1492]: is it now correct?

Comment (ccw808) [reply Client/multiplayer_sa/CMultiplayerSA.cpp:1492]: Yes

**Pull request merged**